apps/nextjs-website/react-components/components/Editorial/Ctas.tsx

```diff
@@ -0,0 +1,90 @@
+import Image from 'next/image';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import appleBadge from '../../../public/editorial-images/app-store-badge.png';
+import googleBadge from '../../../public/editorial-images/google-play-badge.png';
+import { EditorialCtaProps } from '../../types/Editorial/Editorial.types';
+import { CtaButtons } from '../common/Common';
+
+export const Ctas = ({
+  ctaButtons,
+  storeButtons,
+  theme,
+}: EditorialCtaProps) => {
+
+  if (storeButtons?.hrefGoogle || storeButtons?.hrefApple) {
+    return (
+      <Stack
+        sx={{
+          flexDirection: 'column',
+          '@media screen and (min-width: 420px)': {
+            flexDirection: 'row-reverse',
+          },
+        }}
+        justifyContent='left'
+        spacing={2}
+        alignItems='baseline'
+      >
+        {storeButtons.hrefGoogle && (
+          <Button
+            sx={{
+              padding: '0px',
+              marginLeft: '0px',
+              '@media screen and (min-width: 420px)': {
+                marginLeft: '16px',
+              },
+              justifyContent: 'start',
+            }}
+            key='google'
+            href={storeButtons.hrefGoogle}
+          >
+            <Image
+              src={googleBadge}
```

Comment:
How about using the image's base64 encoded string?

It would make react-components more "atomic" and save us the need to add the images to the `public` folder.

Only downside I can see would be not being able to quickly check what the images look like, but I think it's a fair trade-off.